jobsdb/jobsdb.go

```diff
@@ -552,6 +552,7 @@ var (
 	//Valid, Terminal
 	Succeeded   = jobStateT{isValid: true, isTerminal: true, State: "succeeded"}
 	Aborted     = jobStateT{isValid: true, isTerminal: true, State: "aborted"}
+	Drained     = jobStateT{isValid: true, isTerminal: true, State: "drained"}
```

Comment:
We should refrain from introducing new states unless:
1. We have a very good reason
2. We don't use it ad hoc, but make sure we adapt all affected areas 

Please explain the reasoning behind introducing this new state

Reply:
I would agree I didn't have a very strong reason to do that. My only motivation was to be able to identify the jobs which were aborted due to such drainage by a different marker in the reports table. It also made it easy for me to filter out such jobs inside the commitStatusList. I will defer to your judgement on this. In this implementation, we can use other pieces of information such as the reason to filter out the jobs

Reply:
The new state which was added has been removed